### PR TITLE
Approximating site coordinates [BigQuery]

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/data-mgt/python/devices-tranformation/transformation.py [hunk 1]
```diff
@@ -17,6 +17,29 @@ def __init__(self, output_format):
...
+                    updated_sites.append({
+                        "id": site["_id"],
+                        "approximate_latitude": approximated_coordinates["approximate_latitude"],
+                        "approximate_longitude": approximated_coordinates["approximate_longitude"],
+                        "tenant": tenant
+                    })
+            except Exception as ex:
+                print(ex)
+
+        self.airqo_api.update_sites(updated_sites)
```
Comment (NoahNsimbe) [inline src/data-mgt/python/devices-tranformation/transformation.py:41]: functionality to loop through all current devices and attached approximated coordinates. @Baalmart

File: src/airflow/airqo_etl_utils/schema/sites.json [hunk 2]
```diff
@@ -18,12 +18,27 @@
...
+  },
+    {
+    "name": "approximate_latitude",
+    "type": "FLOAT",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "approximate_longitude",
+    "type": "FLOAT",
+    "mode": "NULLABLE"
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/schema/sites.json:30]: Updating the BigQuery Schema.

File: src/data-mgt/python/devices-tranformation/transformation.py (hunk 1 above)
Comment (Baalmart) [reply src/data-mgt/python/devices-tranformation/transformation.py:41]: Alright, looks good to me. Thanks @NoahNsimbe

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**